src/update.rs

```diff
@@ -0,0 +1,17 @@
+//! When wasmer self-update is executed, this is what gets executed
+use std::process::{Command, Stdio};
+use std::io;
+
+pub fn self_update() {
+    println!("Fetching latest installer");
+    let cmd = Command::new("curl").arg("https://get.wasmer.io").arg("-sSfL")
```

Comment:
Since `install.sh` is already in the root dir, Idon't think there is a need to fetch the script again.

Reply:
What if the install script changes?

Reply:
Hmmm

Reply:
Yup, I did that on propose. The install script might evolve over time and I want to make sure we support any future cases that we might not foresee if we bundle `install.sh` inside the package